tests/baselines/reference/tsxAttributeInvalidNames.errors.txt

```diff
@@ -49,5 +49,5 @@ file.tsx(11,20): error TS1161: Unterminated regular expression literal.
 !!! error TS2304: Cannot find name 'data'.
                 ~
 !!! error TS1005: ';' expected.
-                       
+                      ~~

```

Comment:
Should we include angle brackets (in addition to parens and braces) to the logic?

Reply:
With respect to treating them as unterminated? I don't think that's necessary. `(?<)` will still produce an error during the grammar check pass while still maintaining a proper bound for the RegExp body.

Also, Annex B does not treat `/\k</` as an error since the RegExp does not define a named capture group, which indicates that `<>` handling is not a lexical syntax error.

Reply:
I'm not too terribly concerned about this case, to be honest. Parsing falls off the rails here because of malformed JSX, not a malformed RegExp.